ibm/mas_devops/playbooks/oneclick_core.yml

```diff
@@ -16,6 +16,7 @@
     # Workspace Configuration
     mas_workspace_name: "{{ lookup('env', 'MAS_WORKSPACE_NAME') | default('MAS Development', true) }}"
     mas_workspace_id: "{{ lookup('env', 'MAS_WORKSPACE_ID') | default('masdev', true) }}"
+    bas_action: "{{ lookup('env', 'BAS_ACTION') | default('UDS', True) }}"
```

Comment:
@yuvraj-vansure can u bas_action to always be upper case? Incase users set BAS_ACTION to lower case 'uds' or 'dro'

for example
`{{ value | upper }}`

Reply:
updated task to make var always upper case